apps/prairielearn/src/components/QuestionsTable.html.ts

```diff
@@ -59,153 +75,38 @@ export function QuestionsTable({
-              data-filter-control="select"
-              data-filter-control-placeholder="(All Versions)"
-              data-filter-data="func:versionList"
-              data-filter-custom-search="badgeFilterSearch"
-              data-visible="${has_legacy_questions}"
-              data-switchable="true"
-            >
-              Version
-            </th>
-            <th
-              data-field="grading_method"
-              data-sortable="true"
-              data-class="align-middle text-nowrap"
-              data-filter-control="select"
-              data-filter-control-placeholder="(All Methods)"
-              data-visible="false"
-              data-switchable="true"
-            >
-              Grading Method
-            </th>
-            <th
-              data-field="external_grading_image"
-              data-sortable="true"
-              data-class="align-middle text-nowrap"
-              data-filter-control="select"
-              data-filter-control-placeholder="(All Images)"
-              data-visible="false"
-              data-switchable="true"
-            >
-              External Grading Image
-            </th>
-            ${(course_instances || []).map(
-              (course_instance) =>
-                html` <th
-                  data-field="assessments_${course_instance.id}"
-                  data-class="align-middle text-nowrap"
-                  data-formatter="assessments${course_instance.id}Formatter"
-                  data-filter-control="select"
-                  data-filter-control-placeholder="(All Assessments)"
-                  data-filter-data="func:assessments${course_instance.id}List"
-                  data-filter-custom-search="badgeFilterSearch"
-                  data-visible="${current_course_instance &&
-                  idsEqual(current_course_instance.id, course_instance.id)}"
-                  data-switchable="true"
-                >
-                  ${course_instance.short_name} Assessments
-                </th>`,
-            )}
-          </tr>
-        </thead>
-      </table>
+              <i class="fas fa-th-list"></i> Columns
+            </button>
+            <div class="dropdown-menu js-column-visibility"></div>
+          </div>
+          <button class="btn btn-secondary js-clear-filters-btn">
+            <i class="fas fa-times"></i> Clear filters
+          </button>
+          ${showAddQuestionButton
+            ? html`<form class="btn-group" name="add-question-form" method="POST">
+                <input type="hidden" name="__csrf_token" value="${__csrf_token}" />
+                <input type="hidden" name="__action" value="add_question" />
+                <button type="submit" class="btn btn-secondary">
+                  <i class="fas fa-plus"></i> Add Question
+                </button>
+              </form>`
+            : ''}
+        </div>
+      </div>
+      <div id="questionsTable" class="table table-sm table-bordered">
```

Comment:
Let's get rid of Bootstrap's default bottom margin:

```suggestion
      <div id="questionsTable" class="table table-sm table-bordered mb-0">
```

<img width="366" alt="Screenshot 2024-04-17 at 15 00 04" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/64135d2b-3996-4453-8b27-3d6cdf0b3372">